hybrid-blazor-apps.md

```diff
@@ -0,0 +1,100 @@
+position: 270
+---
+
+# UI for Blazor in Hybrid Apps
+
+The [WebView](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-net-6-preview-3/#blazorwebview-controls-for-wpf-windows-forms) feature available as of .NET 6.0 now allows you to embed Blazor components into native MAUI, WPF and WinForms apps.
+
+This article provides details on how to setup the apps to use the Telerik UI for Blazor components.
+
+>tip Explore the [Hybrid Blazor Apps (Blazor Web apps running in WinForms, WPF, MAUI)](https://github.com/telerik/blazor-ui/tree/master/common/hybrid-blazor-apps) sample app.
+
+
+>caption In this article:
+
+* [Prerequisites](#prerequisites)
+* [Add the UI for Blazor components](#add-the-ui-for-blazor-components)
+* [Run the apps](#run-the-apps)
+* [Notes](#notes)
+
+
+## Prerequisites
+
+Prior to adding the Telerik components, you need to make sure you can run the corresponding technology stack and the basic Hybrid Blazor WebView in it.
+
+You need the following prerequisites:
+
+1. Install the latest version of [.NET 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0). It requires [Visual Studio 2022 Preview for Windows](https://visualstudio.microsoft.com/vs/preview/) or for [Mac](https://docs.microsoft.com/en-us/visualstudio/releasenotes/vs2019-mac-preview-relnotes).
+
+1. Install [WebView](https://docs.microsoft.com/en-us/dotnet/maui/user-interface/controls/webview).
+
+1. Make sure you have the necessary bits to work with WinForms/WPF/MAUI apps. For MAUI installation, follow the instructions in the [official documentation](https://docs.microsoft.com/en-us/dotnet/maui/get-started/first-app?pivots=devices-android).
+
+## Add the UI for Blazor components
+
+The process for adding Telerik UI for Blazor in the WinForms/WPF/MAUI app is similar to including the components in a native Blazor app.
+
+#### 1. Get the `Telerik UI for Blazor` package
+
+To use the  UI for Blazor components you need to install the `Telerik.UI.for.Blazor` package and include its reference in the `.csproj` file of the app. [Read more on where to get the `Telerik.UI.for.Blazor` package...]({%slug getting-started/what-you-need%}#get-the-telerik-packages)
+
+#### 2. Add the Telerik client assets
+
+To have the Telerik Blazor components look and behave as expected, you need the Telerik [CSS and JavaScript assets]({%slug getting-started/what-you-need%}#client-assets). Include the assets inside the `<head>` of the `wwwroot/index.html` file.
+
+You may add the Telerik resources as [static assets]({%slug getting-started/what-you-need%}#static-assets) or reference them from a [cloud CDN]({%slug getting-started/what-you-need%}#cdn).
+
+#### 3. Include `@using` statements 
+
+You can set the project to recognize all Telerik components without explicit `@using `statements on every `.razor` file. To achieve this, add the following to your `~/_Imports.razor`file:
+
+````
+@using Telerik.Blazor
+@using Telerik.Blazor.Components
+@using Telerik.Blazor.Services
```

Comment:
This line is not necessary in Blazor apps. Is the addition here on purpose? If yes, we can mention why.